components/client/home/rankings.js

```diff
@@ -38,7 +38,7 @@ export const Rankings = () => {
             className={linkClasses}
             href="https://education.macleans.ca/feature/canadas-best-comprehensive-universities-rankings-2024/"
```

Comment:
This URL seems to redirect. Could you clarify via the support ticket where it should go?